downstairs/src/region.rs

```diff
@@ -1132,7 +1164,7 @@ impl Region {
         let next_eid = self.extents.len() as u32;
 
         let these_extents = (next_eid..self.def.extent_count())
-            .into_par_iter()
+            .into_iter()
```

Comment:
We can probably take rayon out of Cargo.toml now?

Reply:
Right, done in d558cae